tools/metrics/gauges_cf.go

```diff
@@ -52,30 +52,35 @@ func QuotaGauge(c *Client, interval time.Duration) MetricReadCloser {
 				Time:  time.Now(),
 				Name:  "op.quota.services.reserved", // number of services reserved by quotas
 				Value: float64(reservedServices),
+				Unit:  "count",
 			},
 			{
 				Kind:  Gauge,
 				Time:  time.Now(),
 				Name:  "op.quota.services.allocated", // number of services in use
 				Value: float64(allocatedServices),
+				Unit:  "count",
 			},
 			{
 				Kind:  Gauge,
 				Time:  time.Now(),
 				Name:  "op.quota.memory.reserved", // memory reserved by org quotas
 				Value: float64(reservedMemory),
+				Unit:  "megabytes",
 			},
 			{
 				Kind:  Gauge,
 				Time:  time.Now(),
 				Name:  "op.quota.memory.allocated", // memory allocated to apps
 				Value: float64(allocatedMemory),
+				Unit:  "megabytes",
 			},
 			{
 				Kind:  Gauge,
 				Time:  time.Now(),
 				Name:  "op.quota.routes.reserved", // number of routes reserved
 				Value: float64(reservedRoutes),
+				Unit:  "megabytes",
```

Comment:
```suggestion
				Unit:  "routes",
```

or 

```suggestion
				Unit:  "count",
```

... though do we want to get into a metric taxonomy discussion? Probably not :-)

Reply:
This was a mistake, I'll change it to count.

Reply:
fixed